examples/distro does not build

**Describe the bug**
I cannot build the example in this repository.

**Steps to reproduce**
Clone the repository (I used 15e5c2280a30ff9ada027b2efbb80d3cc2ecdfd4) `cd` into `examples/distro` and run `./gradlew build`.

**What did you expect to see?**
A successful build.

**What did you see instead?**
Output with a build failure:

```
> Task :custom:compileJava FAILED
C:\workspaces\otel\opentelemetry-java-instrumentation\examples\distro\custom\src\main\java\com\example\javaagent\DemoSampler.java:22: error: cannot find symbol
      SpanKind spanKind, Attributes attributes, List<LinkData> parentLinks) {
      ^
  symbol:   class SpanKind
  location: class DemoSampler
C:\workspaces\otel\opentelemetry-java-instrumentation\examples\distro\custom\src\main\java\com\example\javaagent\DemoSampler.java:23: error: cannot find symbol
    if (spanKind == SpanKind.INTERNAL && name.contains("greeting")) {
                    ^
  symbol:   variable SpanKind
  location: class DemoSampler
2 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':custom:compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
2 actionable tasks: 2 executed
```

**What version are you using?**
`main` branch at 15e5c2280a30ff9ada027b2efbb80d3cc2ecdfd4.

**Additional context**
It looks like someone updated the DemoSampler.java file without checking it compiles. It seems that this example (which is not contained in the main gradle build) is not built as part of any automatic GitHub workflow?

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -32,6 +32,25 @@ jobs:
           timeout_minutes: 90
           max_attempts: 3
 
+  example-distro:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 11 for running checks
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Restore cache
+        uses: burrunan/gradle-cache-action@v1.6
+        with:
+          job-id: jdk11
+
+      - name: Build
+        run: ./gradlew build --stacktrace
+        working-directory: examples/distro
+
   test:
     runs-on: ubuntu-latest
     strategy:

--- a/.github/workflows/pr.yaml
+++ b/.github/workflows/pr.yaml
@@ -32,6 +32,25 @@ jobs:
           path: /tmp/deadlock-detector-*
           if-no-files-found: ignore
 
+  example-distro:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 11 for running checks
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Restore cache
+        uses: burrunan/gradle-cache-action@v1.6
+        with:
+          job-id: jdk11
+
+      - name: Build
+        run: ./gradlew build --stacktrace
+        working-directory: examples/distro
+
   test:
     runs-on: ubuntu-latest
     strategy:

